libraries/AP_WindVane/AP_WindVane.cpp

```diff
@@ -289,7 +289,7 @@ float AP_WindVane::read_SITL_direction_ef()
     const float wind_speed = AP::sitl()->wind_speed_active;
     const float wind_dir_rad = radians(AP::sitl()->wind_direction_active);
 
-    // Note than the SITL wind direction is defined as the direction the wind is travelling to
+    // Note than the SITL wind direction is defined as the direction the wind is traveling to
```

Comment:
apparently the difference between "traveling" and "travelling" is just a UK vs US thing..  it's fine to change it though.

Reply:
I worked out how to do spellcheck in notepad++ after doing the original sailboat stuff, after you had to correct so much of my bad spelling. This was picked up by the spellchecker, not sure what version of English I have it set to tho